Gemfile

```diff
@@ -20,10 +20,12 @@ gem 'decidim-selectable-news', path: 'decidim-selectable-news'
 gem 'decidim-admin-search_user', path: 'decidim-admin-search_user'
 #### Custom gems and modifciations block end ####
 
+gem 'decidim-idcat_mobil', git: "https://github.com/gencat/decidim-idcat_mobil.git", branch: "master"
```

Comment:
Should we use a tag instead a branch name?
Otherwise, Does this gem have the three locales of Participa gencat? Or should they be created in the participa gencat repository?

Reply:
As per conversation we'll deploy first with branches, check, tag and publish gems and redeploy.

We should add all locales here, thanks for making me notice :D